frontend/src/features/admin-form/create/workflow/components/WorkflowContent/EditStepBlock/RespondentBlock/Components/ConditionalRoutingImageCarousel.tsx

```diff
@@ -0,0 +1,92 @@
+import { useState } from 'react'
+import { FaChevronLeft, FaChevronRight } from 'react-icons/fa'
+import { Button, Image, Stack, Text } from '@chakra-ui/react'
+
+import { ProgressIndicator } from '~components/ProgressIndicator/ProgressIndicator'
+
+interface ImageCarouselProps {
+  images: string[]
+  captions: string[]
+}
+
+export const ImageCarousel = ({ images, captions }: ImageCarouselProps) => {
+  const NUM_STEPS = images.length
```

Comment:
/nit usually `FULL_CAPS` are for magic constants. This seems like it is a regular constants. Would suggest using `numSteps`.